uint8 brightness
--- 
bool success
